psydac/linalg/block.py

```diff
@@ -292,6 +293,20 @@ def __getitem__( self, key ):
     def __setitem__( self, key, value ):
         assert value.space == self.space[key]
         self._blocks[key] = value
+    def conjugate( self, out=None):
+        if out is not None:
+            assert isinstance( out, BlockVector )
+            assert out.space is self.space
+        else:
+            out = BlockVector(self.space)
+        for (Lij, Lij_out) in zip(self.blocks, out.blocks):
+            assert isinstance( Lij, (StencilVector, BlockVector) )
+            assert isinstance( Lij_out, (StencilVector, BlockVector) )
+            Lij.conjugate(out=Lij_out)
+        out._sync = self._sync
+        return out
+    def conj( self, out=None):
```

Comment:
Can you add one empty line between the three functions `__setitem__`, `conjugate` and `conj`?
Also, hoping I finally understood how @yguclu wants his spaces are parentheses ;), we don't use a space after "(" or before ")" in the method header, i.e., write `def conjugate(self, out=None)` instead of `def conjugate( self, out=None)`.
This is confusing at times, as the PSYDAC intern rules have changed in the past but have not been applied everywhere.

Reply:
[6f9f621](https://github.com/pyccel/psydac/pull/273/commits/6f9f621be8e4ab340f6d97b1347bbe6b72560f3b)